Model/Resolver/Invoices.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace ScandiPWA\SalesGraphQl\Model\Resolver;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Sales\Api\Data\OrderInterface;
+use Magento\Sales\Api\Data\InvoiceInterface;
+use Magento\SalesGraphQl\Model\Resolver\Invoices as SourceInvoices;
+
+/**
+ * Resolver for Invoice
+ */
+class Invoices extends SourceInvoices
+{
+    /**
+     * @inheritDoc
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ) {
+        if (!(($value['model'] ?? null) instanceof OrderInterface)) {
+            throw new LocalizedException(__('"model" value should be specified'));
+        }
+
+        /** @var OrderInterface $orderModel */
+        $orderModel = $value['model'];
+        $invoices = [];
+
+        /** @var InvoiceInterface $invoice */
+        foreach ($orderModel->getInvoiceCollection() as $invoice) {
+            $invoices[] = [
+                'id' => base64_encode($invoice->getEntityId()),
+                'number' => $invoice['increment_id'],
+                'comments' => $this->getInvoiceComments($invoice),
+                'model' => $invoice,
+                'order' => $orderModel
+            ];
+        }
+        return $invoices;
+    }
+
+    /**
+     * Get comments invoice in proper format
+     *
+     * @param InvoiceInterface $invoice
+     * @return array
+     */
+    public function getInvoiceComments(InvoiceInterface $invoice): array
+    {
+        $comments = [];
+
+        foreach ($invoice->getComments() as $comment) {
+            if ($comment->getIsVisibleOnFront()) {
```

Comment:
So there was a graphql endpoint for invoice comments but resolver was never returning them from default magento?)

Reply:
Well, at least judging by my local magento files, it is...

Reply:
Yea, also checked, create endpoint in graphql, but it is always null)